src/extractor/ValueSetConceptComponentRuleExtractor.ts

```diff
@@ -0,0 +1,39 @@
+import { fshtypes } from 'fsh-sushi';
+import { cloneDeep } from 'lodash';
+import { ExportableValueSetConceptComponentRule } from '../exportable';
+import { ProcessableValueSetComponent } from '../processor';
+
+export class ValueSetConceptComponentRuleExtractor {
+  static process(
+    vsComponent: ProcessableValueSetComponent,
+    include: boolean
+  ): ExportableValueSetConceptComponentRule {
+    if (vsComponent.concept?.length > 0) {
+      const rule = new ExportableValueSetConceptComponentRule(include);
+      rule.from.system = vsComponent.system;
+      rule.from.valueSets = cloneDeep(vsComponent.valueSet);
```

Comment:
This will usually explicitly set at least one key to `undefined`, resulting in an object like:
```ts
{
  "system": undefined,
  "valueSets": ["http://foo.org/VS"]
}
``` 
or
```ts
{
  "system": http://foo.org/CS,
  "valueSets": undefined
}
```

Sometimes we care about that when we might check objects for existence or keys, but it's probably fine here, right?  I just like to double-check these sorts of things.

Reply:
I'm reasonably confident about this, based on my test behavior and the current implementation of `fromString` in `ExportableValueSetComponentRule.ts`.